sanic/pages/styles/DirectoryPage.css

```diff
@@ -0,0 +1,67 @@
+#breadcrumbs a:hover {
+    text-decoration: underline;
+}
+
+#breadcrumbs .path-0 a {
+    text-decoration: none;
+}
+
+#breadcrumbs span::after {
+    content: "/";
+    text-decoration: none;
+    padding: 0 0.25em;
+}
+
+#breadcrumbs .path-0 a::before {
+    content: "🏠";
+}
+
+#breadcrumbs>span>a {
+    color: #ff0d68;
+}
+
+main a {
+    color: inherit;
+    font-weight: bold;
+}
+
+table.autoindex {
+    width: 100%;
+    font-family: monospace;
+}
+
+table.autoindex tr {
+    display: flex;
+}
+
+table.autoindex tr:hover {
+    background-color: #ddd;
+}
+
+table.autoindex td {
+    margin: 0 0.5rem;
+}
+
+table.autoindex td:first-child {
+    flex: 1;
+}
+
+table.autoindex td:nth-child(2) {
+    text-align: right;
+}
+
+table.autoindex td:last-child {
+    text-align: right;
+}
+
+@media (min-width: 1143px) {
+    table.autoindex {
+        font-size: 1.25rem;
+    }
+}
```

Comment:
Should ideally do these only on base page and via html element font size (that then scales everything). Responsive design done from multiple places gets tricky with potentially conflicting rules. No change request for now, I need to look at this in browser first, and need this comment as a reminder.

Reply:
Removed media query as no longer necessary